portfolio/src/main/webapp/projects.html

```diff
@@ -29,11 +29,14 @@
             </a>
         </div>
 
-
         <!-- my projects -->
         <div>
             <h3>Projects</h3>
-            <h5>Accessible Mouse Design</h5>
+            <!-- load project content from script.js -->
+            <div id = "projectContent"> </div> 
+
+    
+            <!-- <h5>Accessible Mouse Design</h5>
```

Comment:
is this removing all of this content from the page? 

Are you planning on adding it back later? If so, file an issue and link the issue in the comment saying
"TODO(<issue id>) uncomment this when XYZ happens."

If you're not planning to add it back later, just delete it.

Reply:
I wasn't sure if the refactoring is a good idea (it saves about 10 lines of code only) so I wanted to double check with you. 
Should have made this clear in the comment, sorry!